src/components/common/PopoverMenu.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { ListItem, MenuItem, MenuList, Popover, useTheme } from '@mui/material';
+import { DropdownItem } from '@terraware/web-components/components/Dropdown';
+
+type PopoverMenuProps = {
+  items: DropdownItem[] | [];
```

Comment:
why the ` | []`;

Reply:
Because in the org dropdown orgs could be empty

Reply:
DropdownItem[] also captures empty arrays.. maybe I'm missing some TS error you are seeing